swarm/fuse/swarmfs_unix.go

```diff
@@ -120,6 +120,10 @@ func (swarmfs *SwarmFS) Mount(mhash, mountpoint string) (*MountInfo, error) {
 
 	log.Trace("swarmfs mount: traversing manifest map")
 	for suffix, entry := range manifestEntryMap {
+		if suffix == "" { //empty suffix means that the file has no name - i.e. this is the default entry in a manifest. Since we cannot have files without a name, let us ignore this entry
+			log.Warn(fmt.Sprintf("Manifest has a default entry with hash %v. This is ignored and not shown in FUSE", entry.Hash))
```

Comment:
I would change the log text to:

"Manifest has an empty path entry with default entry %v, which would cause FUSE to mount improperly. Ignoring"

Reply:
Then in the comment, please add a bit more descriptive information what happens if this is not done, i.e. what you described in the chat, in some short form. I did not immediately see that this change in fact actually fixes the problem :)

"actually the mounting works, but you only see a few files (random sample) and no directories and an ls command gives an input/output error... but if you do ls subdirectory/ then that works again.
in short, the behaviour is erratic"